src/mixins/date-mixin.ts

```diff
@@ -101,6 +101,28 @@ export default class DateMixin extends Vue {
     return timeStr
   }
 
+  /**
+   * Converts a Date object to a date string (Month Day, Year).
+   * @example "2021-01-01 08:00:00 GMT" -> "Jan 1, 2021"
+   */
+  utcDateToDisplayDate (date: Date): string {
```

Comment:
This method has been replaced by `dateToPacificDate`.

Just a note that the date mixin HERE is to allow these shared components to work in StoryBook. However, once these components are imported into a project, they will use the date mixin THERE.

So, yes, there are 2 version of the date mixin. I hope we will resolve this some time, but atm this is the situation.

Reply:
I got rid of unhelpful date adjectives like "displayDate", "formattedDate", "prettyDate", "uglyDate", etc, etc 😄 

Now we have the following date formats:
- JS Date (object)
- "yyyyMmDd" 
- "apiDate"
- "pacificDate" (and pacificTime and pacificDateTime)

Reply:
This is a new method.
Interesting, i did not know that!
I guess we will need to add this to our Create UI as well 🤔

Reply:
Hmm not sure what to do then. The Datepicker value is `yyyy-mm-dd` and we want to cast that into `Jan 1, 2021` for example just for display puposes. There is no pacific time conversion which why i opted to keep it as utc for simplicity.
In that context, what would be a fitting name?

Reply:
I think* this format is the preferred format for display purposes... at least according to the latest UXA.

Reply:
According to the header for both `dateToPacificDate()` and `utcDateToDisplayDate()`, they do the same thing.

I believe I deleted utcDateToDisplayDate in favour of using dateToPacificDate in multiple places already,

Reply:
> Hmm not sure what to do then. The Datepicker value is yyyy-mm-dd and we want to cast that into Jan 1, 2021 for example just for display puposes. There is no pacific time conversion which why i opted to keep it as utc for simplicity.
> In that context, what would be a fitting name?

I believe what you have in the date picker component is fine (see my latest comments there).

I think you can delete this new method.

Reply:
> I think* this format is the preferred format for display purposes... at least according to the latest UXA.

Yes, I think so too.

Reply:
Updated to use previously existing `yyyyMmDdToPacificDate`, which works great, i missed this method previously i think.

Reply:
Updated!